external-crates/move/move-compiler/src/hlir/translate.rs

```diff
@@ -307,6 +316,9 @@ fn function(context: &mut Context, _name: FunctionName, f: T::Function) -> H::Fu
         body,
     } = f;
     context.env.add_warning_filter_scope(warning_filter.clone());
+    if DEBUG_PRINT {
+        println!("Processing {:?}", _name);
+    }
```

Comment:
Should we remove these before landing?

Also happy to try to standardize some sort of debug printing environment setup

Reply:
I was going to leave this in for the `compiler_dev` branch and remove them all before landing in `main`. I would like to standardize something like this longer-term.